frontend/app/views/error500.scala.html

```diff
@@ -8,9 +8,10 @@
                     Sorry, there was an error on this page. We're aware of the problem. If this error persists, please contact
                     support on <a href="mailto:membershipsupport@@theguardian.com">membershipsupport@@theguardian.com</a>.
                 </p>
-                @ex match { case err: play.api.UsefulException =>
+                @ex match { case err: play.api.UsefulException => {
                     <p>Please provide the following error code to help identify the issue.</p>
                     <pre>Error code: @err.id</pre>
+                    }
```

Comment:
this stopped compiling and this was the solution https://stackoverflow.com/questions/9748258/match-case-in-scala-template-doesnt-work-in-play2